src/component/body/RestaurantList.jsx

```diff
@@ -0,0 +1,31 @@
+import styles from '../../css/Body.module.css';
+
+const RestaurantList = ({ restaurants }) => {
+  return (
+    <section className={styles['restaurant-list-container']}>
+      <ul className={styles['restaurant-list']}>
+        {restaurants.map((restaurant) => (
+          <li key={restaurant.id} className={styles.restaurant}>
+            <div className={styles.restaurant__category}>
+              <img
+                src={restaurant.imgSrc}
+                alt={restaurant.category}
+                className={styles['category-icon']}
+              />
+            </div>
+            <div className={styles.restaurant__info}>
+              <h3 className={`$ {styles["restaurant__name"]} text-subtitle`}>
```

Comment:
```suggestion
              <h3 className={`${styles["restaurant__name"]} text-subtitle`}>
```

아마도 오타겠죠? 다음 주차 미션하실 때 같이 수정해 주시길 바래요!